account_invoice_margin/report/account_invoice_report.py

```diff
@@ -8,11 +8,13 @@ class AccountInvoiceReport(models.Model):
     _inherit = "account.invoice.report"
 
     margin = fields.Float(string='Margin', readonly=True)
+    margin_percent = fields.Float(string='Margin (%)', readonly=True)
 
     def _select(self):
         select_str = super()._select()
-        return '%s, sub.margin' % select_str
+        return '%s, sub.margin, sub.margin_percent' % select_str
 
     def _sub_select(self):
         select_str = super()._sub_select()
-        return '%s, SUM(ail.margin_signed) AS margin' % select_str
+        return '%s, SUM(ail.margin_signed) AS margin, ' \
+               'ai.margin_percent AS margin_percent' % select_str
```

Comment:
This should have as well an aggregation formula (SUM, AVG...)

Reply:
ai.margin_percent or ail.ai.margin_percent will be best for avg?

Reply:
ping @legalsylvain